stan/math/prim/scal/prob/std_normal_lpdf.hpp

```diff
@@ -0,0 +1,75 @@
+#ifndef STAN_MATH_PRIM_SCAL_PROB_STD_NORMAL_LPDF_HPP
+#define STAN_MATH_PRIM_SCAL_PROB_STD_NORMAL_LPDF_HPP
+
+#include <stan/math/prim/scal/meta/is_constant_struct.hpp>
+#include <stan/math/prim/scal/meta/partials_return_type.hpp>
+#include <stan/math/prim/scal/meta/operands_and_partials.hpp>
+#include <stan/math/prim/scal/meta/scalar_seq_view.hpp>
+#include <stan/math/prim/scal/err/check_not_nan.hpp>
+#include <stan/math/prim/scal/fun/constants.hpp>
+#include <stan/math/prim/scal/fun/value_of.hpp>
+#include <stan/math/prim/scal/meta/include_summand.hpp>
+#include <boost/random/normal_distribution.hpp>
+#include <boost/random/variate_generator.hpp>
+#include <string>
+
+namespace stan {
+  namespace math {
+
+    /**
+     * The log of the normal density for the specified scalar(s) given
+     * a mean of 0 and a standard deviation of 1. y can be either
+     * a scalar or a vector.
+     *
+     * <p>The result log probability is defined to be the sum of the
```

Comment:
You don't really need this as it's implied by the first paragraph.  But you can leave it if you think it's not clear enough without it.